src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -68,12 +73,20 @@ def copy(
         raise click.ClickException(
             "Both source and target path are local. This operation is not supported."
         )
+    if is_put and recursive:
+        raise click.ClickException("Recursive for PUT is not supported.")
 
     if is_get:
```

Comment:
Changing order of this `if` can improve readability, WDYT?
```suggestion
    if not is_get:
```

Reply:
I don't see that improvement 🤔 , for me it will not change anything. I added second if for put, WDYT?

Reply:
I like helper methods